pkg/cmd/pipeline/start.go

```diff
@@ -254,7 +254,10 @@ func (opt *startOptions) startPipeline(pipelineStart *v1beta1.Pipeline) error {
 			pr.ObjectMeta.GenerateName = usepr.ObjectMeta.Name + "-"
 		}
 		// Copy over spec from last or previous PipelineRun to use same values for this PipelineRun
+		prStatus := pr.Spec.Status
 		pr.Spec = usepr.Spec
+		// Reapply original status in case PipelineRun used was cancelled
+		pr.Spec.Status = prStatus
```

Comment:
is there any issue with doing it like

```
pr.Spec = usepr.Spec
pr.Spec.Status = ""
```
Is there any specific reason, to keep the new struct and assign again, as we create the pr struct above only.

Reply:
Yes, it appears this works fine. Can change it in both cases.

Reply:
Addressed